src/web/components/Card/components/MediaMeta.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { palette } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+
+import Audio from '@frontend/static/icons/audio.svg';
+import Photo from '@frontend/static/icons/photo.svg';
+import Video from '@frontend/static/icons/video.svg';
+
+type Props = {
+    mediaType: MediaType;
+    pillar: Pillar;
+    mediaDuration?: number;
+};
+
+const iconWrapperStyles = ({ pillar, mediaType }: Props) => css`
+    width: 24px;
+    height: 23px;
+    background-color: ${palette[pillar].main};
+    border-radius: 50%;
+    display: inline-block;
+
+    > svg {
+        width: 14px;
+        height: auto;
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: 6px;
+        display: block;
+        transform: ${mediaType === 'Video' ? `translateY(0.0625rem)` : ``};
+    }
+`;
+
+const durationStyles = ({ pillar }: Props) => css`
+    color: ${palette[pillar].main};
+    ${textSans.xsmall({ fontWeight: `bold` })}
+`;
+
+const wrapperStyles = css`
+    display: flex;
+    align-items: center;
+`;
+
+export function secondsToDuration(secs?: number): string {
+    if (typeof secs === `undefined` || secs === 0) {
+        return ``;
+    }
+    const seconds = Number(secs);
+    const h = Math.floor(seconds / 3600);
+    const m = Math.floor((seconds % 3600) / 60);
+    const s = Math.floor((seconds % 3600) % 60);
```

Comment:
This is equivalent to `const s = Math.floor(seconds % 60);` as 3600 is divisible by 60!